nbconvert/preprocessors/execute.py

```diff
@@ -103,7 +103,18 @@ class ExecutePreprocessor(Preprocessor):
             """
             )
     ).tag(config=True)
-
+    
+    shutdown_kernel = Enum(['graceful', 'immediate', 'now'],
```

Comment:
Sorry to quibble, but I don't like having two values that mean the same thing. I'd keep 'immediate' and get rid of 'now'.